nixos/modules/installer/tools/tools.nix

```diff
@@ -23,7 +23,7 @@ let
     inherit (pkgs) runtimeShell;
     nix = config.nix.package.out;
     path = makeBinPath [
-      pkgs.nixUnstable
```

Comment:
the path actually used to just be `path = makeBinPath [ nixos-enter ];`. I see a `nix =` above which should substitute `@nix@` in the file. 
```
export PATH=@nix@/bin:$PATH
```
There also shouldn't be multiple nix versions in `PATH` https://github.com/NixOS/nix/blob/649d3aaf2481b928120b6ce77d68b1b7c68f69e6/src/nix/doctor.cc#L72. So I think we just need to remove this here entirely.

Reply:
Yeah, I was wondering how that path thing there actually worked and didn't came to a conclusion.